src/blocks/media-block/components/save.js

```diff
@@ -0,0 +1,79 @@
+import { Component } from "@wordpress/element";
+import cls from 'classnames';
+
+export class MediaBlockSaveComponent extends Component{
+
+    render () {
+
+        const { attributes } = this.props;
+
+        const isLinkAvailable = attributes.url !== undefined && attributes.url !== "";
+
+        let titleText = isLinkAvailable ?
+            <a href={ attributes.url } className={"c-block__title-link u-theme--link-hover--dark"}> {attributes.title} </a> :
+            attributes.title;
+
+        const styles = cls(
+            'c-media-block c-block l-grid-wrap l-grid-wrap--4-of-7',
+            {'c-block__row': attributes.alignment === "left"},
+            {'c-block__stacked': attributes.alignment === "center"}
+        );
+
+        const imageStyles = cls(
+            'c-block__image',
+            {'l-grid-item--s--1-col l-grid-item--m--2-col l-grid-item--l--2-col l-grid-item--xl--1-col u-padding--zero--sides': attributes.alignment === "left"},
+            {'l-grid-item--s--5-col l-grid-item--m--4-col l-grid-item--l--3-col l-grid-item--xl--4-col l-grid-item--xxl--3-col': attributes.alignment === "center"}
+        )
+
+        const contentStyles = cls(
+            'c-block__content l-grid-item u-spacing u-color--gray u-border-left--black--at-large u-theme--border-color--darker--left',
+            {'l-grid-item--s--3-col l-grid-item--m--2-col l-grid-item--l--2-col l-grid-item--xl--2-col': attributes.alignment === "left"},
```

Comment:
Replace the classes with the following: `l-grid-item--s--5-col l-grid-item--m--4-col l-grid-item--l--3-col l-grid-item--xl--4-col l-grid-item--xxl--3-col`

Reply:
Do you mean the line for the Central version?

Reply:
What I put was wrong. Please use the notes below.